### ISSUE TITLE
non-critical: missing zero-initialisation of FstatResults

### ISSUE THREAD

**Issue by dbkeitel** [PyFstat/PyFstat#11]
from https://gitlab.aei.uni-hannover.de/GregAshton/PyFstat/issues/19 (20191105):

> if `self.whatToCompute = lalpulsar.FSTATQ_2F` isn't set, F-statistic output will be nasty uninitialised random numbers. It seems we're not initialising the FstatResults struct properly. Not sure what the proper way to do so through SWIG is...

**Comment by dbkeitel** [comment]
Simple hack-ish steps to reproduce:

1. In core.py, method init_computefstatistic_single_point() of the ComputeFstat class, change
```
-        self.whatToCompute = lalpulsar.FSTATQ_2F
+        #self.whatToCompute = lalpulsar.FSTATQ_2F
+        self.whatToCompute += lalpulsar.FSTATQ_ATOMS_PER_DET
```
2. Do
```
from tests import ComputeFstat
test=ComputeFstat()
test.setUpClass()
test.test_run_computefstatistic_single_point_injectSqrtSX()
```
==>
```
13:49 INFO    : Initialising FstatResults
-6.4088294e+34
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/dkeitel/git/sources/PyFstat/tests.py", line 201, in test_run_computefstatistic_single_point_injectSqrtSX
    self.assertTrue(FS > 0.0)
  File "/usr/lib/python3.7/unittest/case.py", line 692, in assertTrue
    raise self.failureException(msg)
AssertionError: False is not true
```

**Comment by dbkeitel** [comment]
This looks like an upstream problem with the SWIG bindings not vetoing access to "illegal" fields of the results structure. I've contacted Karl for it.

In the meantime, this poses no practical problems for PyFstat at all unless one deliberately hacks out the FSTATQ_2F bit from the base ComputeFstat class as done above.

I only noticed this because of an earlier bug where the atoms bit for the transient case was overwriting this bit instead of being added to it, which I've long since fixed.

So this is really just a paranoia-level investigation.
